catalyst_voices/packages/internal/catalyst_voices_view_models/lib/src/treasury/treasury_sections.dart

```diff
@@ -18,4 +21,59 @@ sealed class TreasurySectionStep extends BaseSectionStep {
     super.isEnabled,
     super.isEditable,
   });
+
+  String localizedDesc(BuildContext context) => localizedName(context);
+}
+
+final class SetupCampaignDetailsStep extends TreasurySectionStep {
```

Comment:
This is okey for now but in the end all of those steps will come from template, **probably(!)**, and will be dynamic => none of steps name will be localised (l10n)